events/tests/test_api.py

```diff
@@ -679,6 +680,51 @@ def test_occurrences_query_project_user(
     snapshot.assert_match(executed)
 
 
+OCCURRENCES_COUNTS_QUERY = """
+query Occurrences {
+  occurrences {
+    edges {
+      node {
+        enrolmentCount
+        %s
```

Comment:
Looking at these *_QUERY variables in the codebase none of the others need interpolation to be used. Just thinking whether it could add clarity to the code to name this one differently, e.g. OCCURENCES_COUNT_QUERY_TEMPLATE or something like that because it won't work without string interpolation. Naming the variable to be interpolated might also add clarity.

Reply:
I would create a query getter out of it